Contributing-MD

```diff
@@ -0,0 +1 @@
+This is cotributing file
```

Comment:
Contributing files are awesome for projects, some things to remember:
1. Don't forget to name it CONTRIBUTING.md